src/lib/chat/matrix-client.ts

```diff
@@ -203,7 +203,9 @@ export class MatrixClient {
       return {
         otherMembers,
         memberHistory,
-        totalMembers: room.getInvitedAndJoinedMemberCount(),
+        // Even if a member leaves they stay in the member list so this will provide the correct count
+        // for determining if a conversation is 1 on 1
+        totalMembers: room.getMembers().length,
```

Comment:
This was a regression when implementing sliding sync. I've moved back to the old logic and added the comment from the old logic as well